### ISSUE TITLE
Ventilation du CPU

### ISSUE THREAD

**Issue by grazzinigerard** [LaFibre-info/Freepro#68]
Régulièrement le CPU de le free-box-pro est indiquée en „orange” sur l'interface -> temperature excessive.
Certes on est dans le sud ... mais une meilleure ventilation devrais aider et garantir plus de longévité

**Comment by rmanus** [comment]
Est-ce que le NAS est utilisé lorsqu'elle est en orange ?

**Comment by grazzinigerard** [comment]
> Est-ce que le NAS est utilisé lorsqu'elle est en orange ?

non, mais j'ai malgré tout un disque supplémentaire dans la box (peut-être n'aide pas a la circulation de l'air) ;
assez directement en relation avec la temperature ambiante
